composer.json

```diff
@@ -90,51 +90,50 @@
         "sylius/resource": "^1.10",
         "sylius/resource-bundle": "^1.10",
         "sylius/theme-bundle": "^2.3",
-        "symfony/asset": "^5.4.21 || ^6.0",
-        "symfony/config": "^5.4.21 || ^6.0",
-        "symfony/console": "^5.4.21 || ^6.0",
-        "symfony/dependency-injection": "^5.4.21 || ^6.0",
-        "symfony/deprecation-contracts": "^2.5",
-        "symfony/doctrine-bridge": "^5.4.21 || ^6.0",
-        "symfony/doctrine-messenger": "^5.4.21 || ^6.0",
-        "symfony/event-dispatcher": "^5.4.21 || ^6.0",
-        "symfony/expression-language": "^5.4.21 || ^6.0",
-        "symfony/filesystem": "^5.4.21 || ^6.0",
-        "symfony/finder": "^5.4.21 || ^6.0",
-        "symfony/form": "^5.4.21 || ^6.0",
-        "symfony/framework-bundle": "^5.4.21 || ^6.0",
-        "symfony/http-client": "^5.4.21 || ^6.0",
-        "symfony/http-foundation": "^5.4.21 || ^6.0",
-        "symfony/http-kernel": "^5.4.21 || ^6.0",
-        "symfony/intl": "^5.4.21 || ^6.0",
-        "symfony/mailer": "^5.4.21 || ^6.0",
-        "symfony/messenger": "^5.4.21 || ^6.0",
-        "symfony/monolog-bundle": "^3.5",
-        "symfony/options-resolver": "^5.4.21 || ^6.0",
-        "symfony/password-hasher": "^5.4.21 || ^6.0",
-        "symfony/polyfill-iconv": "^1.17",
-        "symfony/polyfill-intl-icu": "^1.22",
-        "symfony/polyfill-mbstring": "^1.17",
-        "symfony/polyfill-php80": "^1.17",
-        "symfony/process": "^5.4.21 || ^6.0",
-        "symfony/property-access": "^5.4.21 || ^6.0",
-        "symfony/property-info": "^5.4.21 || ^6.0",
-        "symfony/proxy-manager-bridge": "^5.4.21 || ^6.0",
-        "symfony/routing": "^5.4.21 || ^6.0",
-        "symfony/security-bundle": "^5.4.21 || ^6.0",
-        "symfony/security-core": "^5.4.21 || ^6.0",
-        "symfony/security-csrf": "^5.4.21 || ^6.0",
-        "symfony/security-http": "^5.4.21 || ^6.0",
-        "symfony/serializer": "^5.4.21 || ^6.0",
+        "symfony/asset": "^6.3.0",
```

Comment:
Just a note,

some of the `sylius-labs/polyfill-symfony-security` features are SF5 specific and can be scheduled to be dropped as well.

Other SyliusLabas polyfills should checked as well.